README.md

```diff
@@ -4,6 +4,11 @@ A [Visual Studio Code](https://code.visualstudio.com/) [extension](https://marke
 
 The Jupyter Extension includes the Jupyter Keymaps and the Jupyter Notebook Renderers extensions by default. The Jupyter Keymaps extension provides Jupyter-consistent keymaps and the Jupyter Notebook Renderers extension provides renderers for MIME types such as latex, plotly, vega, and the like. Both of these extensions can be disabled or uninstalled.
 
+| Link | Description |
+|------|-------------|
+| [File an issue](https://github.com/microsoft/vscode-jupyter/issues/new/choose) | For reporting problems and suggesting enhancements |
+| [Go to wiki](https://github.com/microsoft/vscode-jupyter/wiki) | For FAQs and learning how to use/debug the extension |
```

Comment:
Not sure this is required. For debugging the extension we have a contributing guide.
For using the extension we have VS Code docs.
I.e. I don't think we need to explicitly call this page out, if we do we should first call out the VS Code docs and contributing guide (the latter is already linked in this readme).
Also we might want to provide a link to the discussions channel .